nemo/backends/pytorch/tutorials/chatbot/modules.py

```diff
@@ -174,26 +133,11 @@ class LuongAttnDecoderRNN(TrainableNM):
     @property
     def input_ports(self):
         """Returns definitions of module input ports.
-
-        targets:
-            0: AxisType(TimeTag)
-
-            1: AxisType(BatchTag)
-
-        encoder_outputs:
-            0: AxisType(TimeTag)
-
-            1: AxisType(BatchTag)
-
-            2: AxisType(ChannelTag)
-
-        max_target_len:
-            None
         """
         return {
-            "targets": NeuralType({0: AxisType(TimeTag), 1: AxisType(BatchTag)}),
-            "encoder_outputs": NeuralType({0: AxisType(TimeTag), 1: AxisType(BatchTag), 2: AxisType(ChannelTag),}),
-            "max_target_len": NeuralType(None),
+            "targets": NeuralType(LabelsType(), ('T', 'B')),
+            "encoder_outputs": NeuralType(ChannelType(), ('T', 'B', 'D')),
+            "max_target_len": NeuralType(axes=None),
```

Comment:
why here no LengthType()?

Reply:
because in this particular case it is a python int object, not a tensor